src/components/SearchInput.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const SearchInput = ({ handleSearch }) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleChange = (e) => {
+    setSearchTerm(e.target.value);
+    handleSearch(e.target.value);
+  };
+
+  return (
+    <div>
+      <input
+        type="text"
+        placeholder="Search by region..."
+        value={searchTerm}
+        onChange={handleChange}
+      />
+    </div>
+  );
+};
+
+SearchInput.propTypes = {
+  handleSearch: PropTypes.func.isRequired,
+};
+
+export default SearchInput;
```

Comment:
As requested by the previous reviewer and as part of the project requirements, kindly make sure that you use a stateless component to filter data retrieved from the API. I can see that this component is not stateless. Kindly update it accordingly. :+1:

Reply:
Thanks